protobuf/model_config.proto

```diff
@@ -130,6 +130,43 @@ message ModelRateLimiter
   uint32 priority = 2;
 }
 
+//@@
+//@@  .. cpp:var:: message SecondaryDevice
+//@@
+//@@     The secondary device to run the model instances on.
+//@@
+message SecondaryDevice
```

Comment:
Can you nest this in ModelInstanceGroup message since it is only used for instance group.